### DISCUSSION TITLE
怎麼避免下雨天腳踏車坐墊濕濕的

### DISCUSSION THREAD

**Discussion by asd99263fg** [1101-GS4719-C/Python-Community-GS4719-C#2013] [General]
如題，不知道各位在中壢有沒有覺得一直下雨很煩，然後每次雨停的時候隔天上課騎腳踏車坐墊都濕濕的，真的很難受，除了用塑膠袋套者還有其他方法解決這個問題嗎

**Comment by jeff110303549** [comment]
完全能夠理解你的想法，我自己也是用一個大一點的塑膠袋繞一圈確保不會有水露到坐墊上，但每一次要拆裝真的有夠麻煩，但我確實沒想過有沒有其他辦法可以解決這個問題，或許這將會成為一個值得思考的專題方向

**Comment by lilishh** [comment]
之前在日本看過不會吸水而且還很柔軟(?)的坐墊套，不過我也是沒有用過，但好像是普遍都蠻推薦的產品，這種好像就不用一直拆裝了！感覺可以找不會吸水甚至滲水的材質試試看。

**Comment by 710388** [comment]
最好的方法就是下雨天不騎車，下雨天騎車超級危險

**Comment by 155Julia** [comment]
我是塑膠的坐墊，每次遇到下完雨騎車我都會直接用手把坐墊上的水撥開，其實差不多就沒有很濕就可以直接坐上去了，但如果太濕也可以隨身攜帶一下抹布，像機車一樣擦一下再騎乘也是一個不錯的方法！

**Comment by yubo0oo** [comment]
我有買一個腳踏車防水墊，才100元而已

**Comment by poyen0806** [comment]
完全能夠理解你的想法，我之前是用一個大一點的塑膠袋繞一圈確保不會有水露到坐墊上。
但現在，我已經超越了，把腳踏車坐墊拆掉就不會有這個問題。
小孩子才需要坐墊，真男人都直接騎。

**Comment by lichilun** [comment]
為了避免下雨天腳踏車坐墊濕濕的，我覺得使用防水坐墊套或選擇防水的坐墊是最簡單又有效的方法。這樣即使遇到小雨也能保持乾爽。如果沒有這些設備，還可以在停車時用塑膠袋覆蓋坐墊，或者找個遮蔽處將腳踏車停好，避免雨水直接打在坐墊上。我也會隨身攜帶毛巾或乾布，這樣可以隨時擦乾坐墊，讓騎行更加舒適。

**Comment by yurukung** [comment]
買一頂帽子 這樣就會冒冒失失的

**Reply by Zheyuan987** [reply]
這笑話很有趣

**Comment by Kaz9487** [comment]
可以隨身攜帶一條抹布或毛巾，要騎車的時候再擦乾就好。不然停在有遮蔽處的地方也是好的解法，ex:工二上課可以停在教研大樓

**Comment by BOSENCHANG2004** [comment]
我自己是會先提早幾天看天氣預報，確定哪幾天比較有可能會下雨，那幾天我就盡量避免騎腳踏車上課，提早幾分鐘出門走。
有幾個好處:
1. 不會坐到濕坐墊
2. 多運動好處多
3. 天與路滑避免騎車也比較安全
希望有解決到同學的疑問!!!

**Comment by a414414** [comment]
我覺得可以在知道過幾天可能會下雨的情況下，事先在腳踏車坐墊上套上塑膠袋，不過套塑膠袋可能不夠環保，如果你不重複用的話。
或是你也可以隨身攜帶一個抹布，反正他只是腳踏車坐墊而已，用一條抹布應該就可以擦乾。
這兩個方法應該都可以避免腳踏車坐墊濕濕的，坐上去褲子濕掉的問題。

**Comment by lilipop6405** [comment]
我在車上有一個小掛袋，裡面有塑膠袋，如果昨天下雨濕濕的話，我會將塑膠袋套在座墊上，這樣就可以防止褲子形成「屁痕」，比較雅觀。我也有同學直接換坐墊，但我覺得我現在這個還沒壞掉就不換了，總覺得還沒壞就換有點浪費錢。

**Comment by SendaiHatsuki** [comment]
這是搞笑一點的方法啦，為了不讓褲子濕有些人會直接乾脆就站著踩車騎，尤其是從宿舍到教室那種短距離，雖然有點荒謬但勉強可以救場，我有時候也會這樣。

**Comment by mattwang1222** [comment]
通常戶先買個防水屁墊，再不行的話加一層精緻塑膠袋綁在坐墊上。下雨過後翻個面就可以得到乾的坐墊

**Comment by yuuu3957** [comment]
我自己是準備抹布，要騎之前擦乾就行了，但就是不會擦得非常乾，坐上去感覺還是會有一點水。

**Comment by shrshu38** [comment]
我都是在車籃裡放一塊防水布或速乾毛巾, 下雨時蓋住坐墊,雨停後掀起來就能騎,屁股不會濕,也不用每次都找塑膠袋套。
